core/templates/dev/head/pages/exploration_player/LearnerViewBreadcrumb.js

```diff
@@ -27,7 +27,7 @@ oppia.controller('LearnerViewBreadcrumb', [
       if (expInfo) {
         openInformationCardModal();
       } else {
-        $http.get('/explorationsummarieshandler/data', {
+        $http.get(EXPLORATION_SUMMARY_DATA_URL_TEMPLATE, {
```

Comment:
Optional: Maybe worth changing this to use ExplorationSummaryBackendApiService, too?